### PR TITLE
Adapting the styling of days cells to React Native new percentage model

### PR DESCRIPTION
**Pull request by cherniv**
Now when RN [supports percents](https://github.com/facebook/react-native/commit/3f49e743bea730907066677c7cbfbb1260677d11), seems like will be better using percents for days width calculation instead of DEVICE_WIDTH/7 calculation. Also it will work fine on Web (through react-native-web) and potentially might solve only-portrait-mode limitation and allow the Lanscape mode as well.

### CODE REVIEW

File: components/styles.js [hunk 1]
```diff
@@ -48,13 +48,13 @@ const styles = StyleSheet.create({
   dayButton: {
     alignItems: 'center',
     padding: 5,
-    width: DEVICE_WIDTH / 7,
+    width: 14.2 + '%',
```
Comment (christopherdro) [inline components/styles.js]: Let's make this a bit more accurate by using 14.2857

File: components/styles.js [hunk 2]
```diff
@@ -48,13 +48,13 @@ const styles = StyleSheet.create({
...
     borderTopWidth: 1,
     borderTopColor: '#e9e9e9',
   },
   dayButtonFiller: {
     padding: 5,
-    width: DEVICE_WIDTH / 7,
+    width: 14.2 + '%',
```
Comment (christopherdro) [inline components/styles.js]: See comment above

File: components/styles.js (hunk 1 above)
Comment (cherniv) [reply components/styles.js]: @christopherdro Done

File: components/styles.js (hunk 2 above)
Comment (cherniv) [reply components/styles.js]: @christopherdro Done

**Pull request merged**